sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveParquetSuite.scala

```diff
@@ -92,4 +92,18 @@ class HiveParquetSuite extends QueryTest with ParquetTest with TestHiveSingleton
       }
     }
   }
+
+  test("SPARK-25271: write empty map into hive parquet table") {
```

Comment:
Although this is the original reported test case in SPARK-25271 `Creating parquet table with all the column null throws exception`, this PR is aiming to resolve the root cause for general issues `Hive ctas commands should use data source if it is convertible`. Can we have more additional test cases outside `HiveParquetSuite`?

Reply:
+1